content/2020/2020-10/2020-10-30-pulsar-weekly.md

```diff
@@ -0,0 +1,137 @@
+
+This is the Pulsar community weekly update for 2020-10-24 ~ 2020-10-30, with updates on Pulsar client, broker, transaction, tiered storage, and so on.
+
+### Pulsar Highlight
+
+The Pulsar Java client supports both `AvroBased` and `UnAvroBased` schemas. For details, see [PR-8246](https://github.com/apache/pulsar/pull/8246).
+
+Before this PR is merged, the Pulsar Java client only supports the `AvroBased` schema.
+
+by [@hnail](https://github.com/hnail))
+
+### Notable Feature
+
+- [Websocket] Update produce request and acknowledgement for WebSocket.
+
+    https://github.com/apache/pulsar/pull/8401 ([@MarvinCai](https://github.com/MarvinCai))
+
+- [Broker] Add the `nettyMaxFrameSizeBytes` setting to the `standalone.conf` file.
+
+    https://github.com/apache/pulsar/pull/8385 ([@massakam](https://github.com/massakam))
+    
+- [Java Client] Disable the batch receiving timer for Readers.
+
+    https://github.com/apache/pulsar/pull/8381 ([@lhotari](https://github.com/lhotari))
+
+- [Broker] Allow the tenant Admin to loo kup topics.
+
+    https://github.com/apache/pulsar/pull/8353 ([@rdhabalia]https://github.com/rdhabalia)
+    
+- [Broker] Implement the `TransactionBuffer` Refactor.
+
+    https://github.com/apache/pulsar/pull/8347 ([@gaoran10]https://github.com/gaoran10)
+
+- [Go Functions] Add the `newOuputMessage` interface for Go Function.
+
+    https://github.com/apache/pulsar/pull/8327 ([@wolfstudy](https://github.com/wolfstudy))
+
+- [Java client] Support cancelling message and batch futures returned from the reader and consumer.
+
+    https://github.com/apache/pulsar/pull/8326 ([@lhotari](https://github.com/lhotari))
+    
+- [Broker] Delete the transaction marker for topics.
+
+    https://github.com/apache/pulsar/pull/8318 ([@congbobo184](https://github.com/congbobo184))
+    
+- [Java client] Support the `UnAvroBased` Schema.
+
+    https://github.com/apache/pulsar/pull/8246 ([@hnail](https://github.com/hnail))
+    
+- [Broker] Supporting deleting associated ledgers before deleting cluster metadata.
+
+    https://github.com/apache/pulsar/pull/8244 ([@BewareMyPower]https://github.com/BewareMyPower)
+    
+- [Broker] Add broker configurations to enforce producer to publish encrypted messages.
+
+    https://github.com/apache/pulsar/pull/8055 ([@rdhabalia](https://github.com/rdhabalia))
+    
+- [Admin API] Support looking up for partitioned topics.
+
+    https://github.com/apache/pulsar/pull/7605 ([@aloyszhang](https://github.com/aloyszhang))
+    
+- [Broker] Support configuring the maximum concurrent HTTP Web requests.
+
+    https://github.com/apache/pulsar/pull/7250 ([@rdhabalia](https://github.com/rdhabalia))
+
+### Notable Bug Fix
+
+- [Broker] Fix the deadlock that occurred during topic ownership check.
+
+    https://github.com/apache/pulsar/pull/8406 ([@massakam](https://github.com/massakam))
+
+- [Java client] Fix the issue that newly-created consumers cannot be paused for subscribing topics.
```

Comment:
It seems that it's not the correct meaning of [this issue](https://github.com/apache/pulsar/issues/8214). If you need help, we can work on the description together.